docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -309,16 +312,71 @@
 
       when: helmv3_installed and helmv3_dm_exists.rc == 0
 
-    - name: Install Deployment Manager
-      shell: KUBECONFIG=/etc/kubernetes/admin.conf /usr/sbin/helm upgrade --install deployment-manager {% if helm_chart_overrides is defined %}--values {{ helm_chart_overrides }}{% endif %} {{ manager_chart }}
-      when: helmv3_installed
```

Comment:
Need to re-visit all the tasks with "helmv3_installed" if the application no longer expected to be exist in helmv3 cli

Reply:
Also request to clean helmv2_installed as no longer supported